src/opentime/timeRange.h

```diff
@@ -5,37 +5,46 @@
 #include <algorithm>
 #include <string>
 
-namespace opentime { namespace OPENTIME_VERSION {
-    
+namespace opentime {
+    namespace OPENTIME_VERSION {
+
+/**It is possible to construct TimeRange object with a negative duration.
+ * However, the logical predicates are written as if duration is positive,
+ * and have undefined behavior for negative durations.
+ *
+ * The duration on a TimeRange indicates a time range that is inclusive of the start time,
+ * and exclusive of the end time. All of the predicates are computed accordingly.
+ */
+
 class TimeRange {
 public:
-    explicit TimeRange() : _start_time {}, _duration {} {}
+    explicit TimeRange() : _start_time{}, _duration{} {}
 
     explicit TimeRange(RationalTime start_time)
-    : _start_time {start_time}, _duration { RationalTime {0, start_time.rate()} } {}
+            : _start_time{start_time}, _duration{RationalTime{0, start_time.rate()}} {}
 
     explicit TimeRange(RationalTime start_time, RationalTime duration)
-    : _start_time {start_time}, _duration {duration} {}
+            : _start_time{start_time}, _duration{duration} {}
+
+    TimeRange(TimeRange const &) = default;
 
-    TimeRange(TimeRange const&) = default;
-    TimeRange& operator= (TimeRange const&) = default;
+    TimeRange &operator=(TimeRange const &) = default;
 
-    RationalTime const& start_time() const {
+    RationalTime const &start_time() const {
         return _start_time;
     }
 
-    RationalTime const& duration() const {
+    RationalTime const &duration() const {
```

Comment:
Did these ampersands move because of the clang format?  Could we not include reformatting with this PR?  It makes it harder to parse the contents specific to this PR vs reformatting the file in general.  Sorry about that, let me know if you need help with git commands to untangle this.

Reply:
Yes, this must be because of clang format. Some help with git here would be great! I've been wondering how to make such fixes.

Reply:
Don't worry about it, should be fine to include this.